src/components/quickStarts.stories.mdx

```diff
@@ -509,10 +509,10 @@ const MyApp = () => {
   const joinMeeting = async () => {
     const response = await fetch('/my-meetings-endpoint');
     const data = await response.json();
+    const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
     const joinData = {
-      meetingInfo: data.Meeting,
-      attendeeInfo: data.Attendee,
-      eventReporter: new NoOpEventReporter();
+      meetingSessionConfiguration: meetingSessionConfiguration,
```

Comment:
same here, initialize the `eventController` as a variable first and use it later would be better.